src/components/HeaderItems/SaveButton.jsx

```diff
@@ -1,71 +1,14 @@
-import { useMutation, useQueryClient } from "@tanstack/react-query";
 import { useAtom, useAtomValue } from "jotai";
 
-import {
-  isEditModeAtom,
-  isRelationshipAtom,
-  userAtom,
-  currentDBIdAtom,
-  changedDocAtom,
-  isListViewAtom,
-  docDataAtom,
-  currentDocIndexAtom,
-  relationshipsDataAtom,
-} from "../../atoms/atoms";
-
-import fetchData from "../../utils/axios";
+import { isEditModeAtom, isRelationshipAtom } from "../../atoms/atoms";
+import usePostSaveChangedData from "../../apis/usePostSaveChangedData";
```

Comment:
만약 커스텀 훅 네이밍을 변경한다면, 함께 변경되어야 하는 부분입니다.

```suggestion
import usePutSaveChangedData from "../../apis/usePutSaveChangedData";
```